### ISSUE TITLE
No thumbnail refresh

### ISSUE THREAD

**Issue by risoul** [getgrav/grav-plugin-admin#828]
Using Landio sketleton, after deleting picture on page media and replace them with others, then saving. The good pictures are showing on front end, but thumbnails are the old ones.

**Comment by w00fz** [comment]
Have you tried clearing all cache?

**Comment by risoul** [comment]
No, but today all is fine. It was just wrong thumbnails just after saving.
